_pkgdown.yml

```diff
@@ -2,6 +2,7 @@ url: https://docs.ropensci.org/rotemplate
 
 template:
   package: rotemplate
+  light-switch: true
```

Comment:
I checked that if this is not included, then despite the navbar default configuration that includes lightswitch on the right, no lightswitch appears on the website.